woocommerce/woocommerce-gutenberg-products-block
# Fix Price Block Inserter Visibility

This change allows for the `Product Pricing` block to be added as a child of the `Post Template` block via the inserter (see example video below).

Additionally, the unused `save.tsx` file is removed.

### Screenshots



https://user-images.githubusercontent.com/481776/199312119-0524a3db-b59c-419c-a4ee-c3b863f9fe1e.mp4

### Testing



1. Checkout PR branch and run `npm start`.
2. Add the Product Query block in a post/page.
3. Confirm it is possible to add the Product Pricing block as a child of Post Template.
4. Confirm it isn't possible to add the Product Pricing block as the main block.
5. Confirm there are no related JS/PHP errors.


### WooCommerce Visibility
